repository/gogit_test.go

```diff
@@ -66,3 +66,36 @@ func TestNewGoGitRepo(t *testing.T) {
 func TestGoGitRepo(t *testing.T) {
 	RepoTest(t, CreateGoGitTestRepo, CleanupTestRepos)
 }
+
+func TestGoGitRepo_Indexes(t *testing.T) {
+	t.Parallel()
```

Comment:
@MichaelMure - I'm pretty sure the issue on Windows is related to this `t.Parallel`.  Is there any interest in running tests concurrently or should I just remove it?

Reply:
I suspect the problems comes from Bleve (the indexing thing) not being closed before attempting to remove the repo from disk. It seems to be a problem in other tests as well.

Reply:
That is, `repo.Close()` should be called before removing the folder on disk. `EraseFromDisk()` would also work.

Reply:
I'm writing tests for the `commands` package and found that this morning too!  I don't have a PR ready (this is to diagnose #778) but the branch is https://github.com/selesy/git-bug/tree/fix/778-git-bug-rm-broken.